src/components/ToDoForm.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <div class="todo-form">
+        <input v-model="currentItem"/>
+        <button @click="add()">Add item</button>
+    </div>
```

Comment:
If you make this a `<form>` element you can add the `required` attribute to the input. This way we can't accidentally add empty items.

Reply:
Since this isn't inside a form tag the required attribute doesn't work but I added a small piece of code to verify that the input isn't empty in the ToDoContainer component lines 26-29.